.github/workflows/ci.yml

```
@@ -44,6 +44,12 @@ jobs:
           gcc_version=$(${LMI_TRIPLET}-${LMI_COMPILER} -dumpversion|tr -d '\r')
           echo "::set-env name=gcc_version::$gcc_version"
 
+          xml2_sha1=$(git rev-parse HEAD:third_party/libxml2)
+          xslt_sha1=$(git rev-parse HEAD:third_party/libxslt)
+          echo "$xml2_sha1 $xslt_sha1" > submodules.hash
+          libs_hash=${{ hashFiles('install_libxml2_libxslt.sh', 'install_wx.sh', 'install_wxpdfdoc.sh', 'submodules.hash') }}
```

Couldn't we just do:
```suggestion
          libs_hash=${{ hashFiles('install_libxml2_libxslt.sh', 'install_wx.sh', 'install_wxpdfdoc.sh') }}-$(git rev-parse HEAD:third_party/libxml2)-$(git rev-parse HEAD:third_party/libxslt)
```
(and then maybe not bother with the env variables at all)?

BTW, we're not supposed to use `::set-env` since a couple of days any longer, now we should [use `GITHUB_ENV` instead](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable). Changing this is not urgent but please make another PR later to do it.

---

See https://github.com/vadz/lmi/pull/155.

---

```
$(git rev-parse HEAD:third_party/libxml2)
```
Looks like evaluating the bash command inside the literal doesn't supported:
https://stackoverflow.com/questions/57968497/how-do-i-set-an-env-var-with-a-bash-expression-in-github-actions